model-optimizer/extensions/front/tf/mvn.py

```diff
@@ -35,7 +35,7 @@ def pattern(self):
                 ('variance', dict(op='ReduceMean')),
                 ('squeeze_mean', dict(op='Squeeze')),
                 ('squeeze_variance', dict(op='Squeeze')),
-                ('fbn', dict(op=lambda op: op in ['FusedBatchNorm', 'FusedBatchNormV2', 'FusedBatchNormV3'])),
+                ('fbn', dict(op=lambda op: op in ['batchNormInference'])),
```

Comment:
I think `batchNormInference` must be capitalized

Reply:
done